cellprofiler/modules/resize.py

```diff
@@ -73,8 +72,30 @@ def create_settings(self):
 -  *Resize by specifying desired final dimensions:* Enter the new height and width of the resized image, in units of pixels.""",
         )
 
-        self.resizing_factor = Float(
-            "Resizing factor",
+        self.resizing_factor_x = Float(
+            "Resizing factor X",
```

Comment:
Personally I prefer dimension first- X Resizing factor rather than Resizing factor x - do you mind asking the group for a consensus?